compiler/rustc_target/src/target_features.rs

```diff
@@ -606,9 +614,19 @@ impl super::spec::Target {
     // Returns None if we do not support ABI checks on the given target yet.
     pub fn features_for_correct_vector_abi(&self) -> Option<&'static [(u64, &'static str)]> {
         match &*self.arch {
+            // Tier 1
```

Comment:
This is not important.
```suggestion
```

Reply:
Since we are still in the process of adding more architectures, and we are proceeding tier-by-tier, it seems useful to document how far we got so far?

Reply:
that is provided on line 629

Reply:
If it is strongly desired I suppose it's fine, I just would prefer not to enshrine our support tiers in such internal commentary. It is prone to falling out of date, if nothing else.

Reply:
Fair point, removed.